android/app/src/main/java/com/github/quarck/calnotify/ui/TestActivity.kt

```diff
@@ -267,4 +265,8 @@ class TestActivity : Activity() {
     fun OnButtonDisableDevPage(v: View) {
         Settings(this).devModeEnabled = false
     }
+    
```

Comment:
do we need this to build? rememeber we want to change as little as possible until things are more througly testsed